src/main/java/org/craftercms/engine/service/context/SiteContextFactory.java

```diff
@@ -451,6 +454,25 @@ public SiteContext createContext(String siteName) {
         }
     }
 
+    /**
+     * Resolve root folder path
+     * @param siteName site name
+     * @return the root folder absolute path
+     */
+    public String resolvedRootFolderPath(String siteName) throws URISyntaxException {
```

Comment:
Why `resolved` (vs `resolve`) ?
`resolvedRootFolderPath`
`resolveRootFolderPath`

Reply:
Updated to use `resolve`. Thank you!